### PR TITLE
Codestyle improvements, add WebDriverNavigationInterface

### PR DESCRIPTION
**Pull request by OndraM**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ This project versioning adheres to [Semantic Versioning](http://semver.org/).
 - Allow installation of Symfony 5 components.
 - Rename environment variable used to pass path to ChromeDriver executable from `webdriver.chrome.driver` to `WEBDRIVER_CHROME_DRIVER`. However the old one also still works to keep backward compatibility
 - If subdirectories in a path to screenshot destination does not exists (using `takeScreenshot()` or `takeElementScreenshot()` methods), they are automatically created.
+- When zip archive cannot be crated during file upload, throw exception instead of silently returning false.
```
Comment (williamdes) [inline CHANGELOG.md]: Typo

Comment (OndraM) [reply CHANGELOG.md]: Thanks

Comment (williamdes) [reply CHANGELOG.md]: @OndraM you merged the typo!
"crted"

Comment (OndraM) [reply CHANGELOG.md]: :man_facepalming: Oh, good you noticed. I forgot to push the change before merging :roll_eyes: . I will fix it, thanks.

**Pull request merged**